syntax = "proto3";
import "currency_pair.proto";

message L2OrderBookEntry {
  string price = 1;
  string size = 2;
  int32 num_orders = 3;
}

message L2OrderBookConfig {
 CurrencyPair product = 1;
}

message L2OrderBook {
  CurrencyPair product = 3;
  repeated L2OrderBookEntry bids = 1;
  repeated L2OrderBookEntry asks = 2;
}
